helm/scylla/templates/rbac.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 # Role for scylla members.
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
+kind: Role
```

Comment:
@zimnx I also ran into an issue with the `ClusterRole`. I want to deploy multiple Scylla Clusters on the same k8s cluster. However, using the `ClusterRole` which receives the same name prevents me from doing so.
I believe a normal Role and RoleBinding are more appropriate for the Scylla deployment, any thoughts?

Reply:
nvm, I've used `fullnameOverride` to overcome that problem, also the change didn't seem to work. So reverting.